cfn/README-GD.md

```diff
@@ -69,34 +70,16 @@ An example of a successful response is:
 
 Make a note of the `access_key_id` and `secret_key` values, which you need to deploy the CloudFormation template to your AWS account. 
 
-**Necessary role error**
-
-If the command returns an error about not having the necessary role, please verify your Alert Logic account has administrator permissions. Click [here](https://console.cloudinsight.alertlogic.com/api/aims/) for more information about AIMS APIs.
+#### To create a key through Windows PowerShell:
 
-**"Limit exceeded" error**
```

Comment:
I'd suggest to move it into some `Troubleshooting` section instead of removing it entirely.

Reply:
James Hastings suggested we remove this passage, because the chances of a customer exceeding the key limit is so incredibly small, it has never happened.

Reply:
Chances are small because we currently provide a few collectors which use AIMs keys. As we add more some users could find that useful.
Let's have it from the beginning  rather than adding it back upon user request.